examples/SimplePulleys/ChineseYoYo.m

```diff
@@ -0,0 +1,153 @@
+classdef ChineseYoYo < HybridDrakeSystem
+  
+  properties
+    in_contact
+    no_contact
+  end
+  
+  methods
+    function obj = ChineseYoYo
+      in_contact = PlanarRigidBodyManipulator('tension.urdf');
+
+      in_contact = PlanarRigidBodyManipulator('tension.urdf');
+
+      % manually remove the ball from the pulley system:
+      pulley_constraint = in_contact.position_constraints{1};
+      cable_length_fcn = pulley_constraint.fcn;
+      cable_length_fcn.pulley(2)=[];
+      pulley_constraint = DrakeFunctionConstraint(pulley_constraint.lb, pulley_constraint.ub, cable_length_fcn);
+      no_contact = in_contact.updatePositionEqualityConstraint(1,pulley_constraint);
+      
+      obj = obj@HybridDrakeSystem(0,1+getNumOutputs(in_contact));
+      obj = setInputFrame(obj,getInputFrame(in_contact));
+      obj = setOutputFrame(obj,MultiCoordinateFrame.constructFrame({CoordinateFrame('mode',1,'m',{'m'}),getOutputFrame(in_contact)}));
```

Comment:
A comment on this frame construction would be helpful, since I'm not sure what it's doing (and why)